riddc/notebooks/fox-kemper/Volume_viewer_converter.ipynb

```diff
@@ -0,0 +1,355 @@
+{
```

Comment:
<p>Separate into sections</p><p> </p><h1>1- Download Volume Viewer</h1><p>text</p><p> </p><h1>2- Mount Google Drive</h1><p> </p><p>Also the link to download volume viewer is not self-explanatory. There is a zip in there. What should pleople do? Get executables, instructions to Oscar?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ridatadiscoverycenter/riddc-jbook/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>done</p>